api/api/specs/jobs.yaml

```diff
@@ -160,6 +160,8 @@ components:
           $ref: "#/components/schemas/EnsemblerJobStatus"
         error:
           type: "string"
+        dashboard_url:
```

Comment:
This `EnsemblingJob` is also used as the request body for the create job request. But we don't expect the dashboard URL to be input (the same problem also exists with the `id` field and perhaps others in the create request).

I think we can define an explicit type to handle the create request. Eg:

```
EnsemblingJob:
      allOf:
        - $ref: '#/components/schemas/EnsemblingJobCreateRequestBody'
        - type: object
          properties:
            id:
              $ref: "#/components/schemas/Id"
            dashboard_url:
              type: string
```

wdyt ?

Reply:
Seems like this problem was solved using the `readOnly: true`, will do that instead. example: https://github.com/ashwinath/turing/blob/f2ea45af2f49114aff304be2585ba6095fb8d3bd/api/api/specs/routers.yaml#L414

Reply:
Nice. TIL. :)